### ISSUE TITLE
[Feature requets] attache and manage previously created VMs

### ISSUE THREAD

**Issue by xavier8854** [DaVieS007/bhyve-webadmin#39]
It could be nice to be able to attache an manage previously created VMs (with vm-bhyve, for example) to BVCP

**Comment by runtimevic** [comment]
Yes, it would be great to be able to import or give it the path of other virtual machines previously created with bhyve...
